charts/opentelemetry-demo/values.yaml

```diff
@@ -527,15 +527,15 @@ components:
     enabled: true
     imageOverride:
       repository: "ghcr.io/open-feature/flagd"
-      tag: "v0.9.0"
+      tag: "v0.10.1"
```

Comment:
can you mention the tag bump in the description?

Reply:
Done